### PR TITLE
Option for counting assertions.

### PR DESCRIPTION
**Pull request by aehyvari**

### CODE REVIEW

File: scrambler.cpp [hunk 1]
```diff
@@ -863,7 +868,13 @@ void usage(const char *program)
               << "    -support-z3 [true|false]\n"
               << "        controls whether to support non-SMTLIB commands that "
                  "are supported\n"
-              << "        by Z3 (default: false)\n";
+              << "        by Z3 (default: false)\n"
+              << "        controls whether the output is in a format suitable for the model\n"
```
Comment (jhoenicke) [inline scrambler.cpp]: this and the next line should be removed.

File: scrambler.cpp [hunk 2]
```diff
@@ -944,6 +955,13 @@ int main(int argc, char **argv)
                 support_z3 = true;
             } else if (strcmp(argv[i + 1], "false") == 0) {
                 support_z3 = false;
+            }
```
Comment (jhoenicke) [inline scrambler.cpp:958]: the `else { usage(arv[0))` is missing here.

**Review by jhoenicke — CHANGES_REQUESTED** [review]
Two small problems, which are easy to fix :)

File: scrambler.cpp [hunk 3]
```diff
@@ -863,7 +868,13 @@ void usage(const char *program)
               << "    -support-z3 [true|false]\n"
               << "        controls whether to support non-SMTLIB commands that "
                  "are supported\n"
-              << "        by Z3 (default: false)\n";
+              << "        by Z3 (default: false)\n"
+              << "        controls whether the output is in a format suitable for the model\n"
+              << "        track of SMT-COMP (default: false)\n"
+              << "\n"
+              << "    -count-asrts [true|false]\n"
```
Comment (HanielB) [inline scrambler.cpp]: can you just write "-count-asserts"? It's more clear and not much longer

**Review by HanielB — APPROVED** [review]
Minor comment, otherwise LGTM

**Comment by jhoenicke** [comment]
The comment at the beginning of `process.assertion-count` also needs to be updated.

**Comment by aehyvari** [comment]
> The comment at the beginning of `process.assertion-count` also needs to be updated.

Yes... I'm thinking of the following:

`This script allows the assertion counting feature of the scrambler to be tested in the regression checking framework.`

File: scrambler.cpp (hunk 3 above)
Comment (aehyvari) [reply scrambler.cpp]: Should be done now.

File: scrambler.cpp (hunk 2 above)
Comment (aehyvari) [reply scrambler.cpp:958]: Done.

File: scrambler.cpp (hunk 1 above)
Comment (aehyvari) [reply scrambler.cpp]: Done

**Review by jhoenicke — APPROVED** [review]

**Pull request merged**